sound/soc/sof/topology.c

```diff
@@ -2815,14 +2815,14 @@ static int sof_set_dai_config(struct snd_sof_dev *sdev, u32 size,
 			      struct snd_soc_dai_link *link,
 			      struct sof_ipc_dai_config *config)
 {
```

Comment:
don't comments in this function - above and below have to be changed now to use our "updated definitions" too?